### PR TITLE
Improve url encoding and add encode option

### LINKED ISSUE #8: base64 transform
It would be good to add base64 support in output
http://codepen.io/Tigt/post/optimizing-svgs-in-data-uris

### PR DESCRIPTION
**Pull request by TrySound**
Fix #8 
I didn't mean base64. Here just lighter version of encoding which still work in iiieee, but smaller :)

### CODE REVIEW

**Comment by TrySound** [comment]
@ben-eb What's happend

**Comment by TrySound** [comment]
I'd like to see this option enabled by default. Or even don't use option and encode everytime. Caz ie fix.

**Comment by TrySound** [comment]
Weird. Tests passes even on windows, but not here.

File: src/index.js [hunk 1]
```diff
@@ -54,13 +56,14 @@ function minifyPromise (svgo, decl) {
 }
 
 module.exports = postcss.plugin('postcss-svgo', opts => {
```
Comment (ben-eb) [inline src/index.js]: Is this better?

```js
module.exports = postcss.plugin('postcss-svgo', (opts = {}) => {
```

**Pull request merged**